agent_application/example.config.yaml

```diff
@@ -78,10 +78,5 @@ did_document_cache:
 
 # Key configuration (temporary)
 secret_manager:
-  stronghold_path: "/tmp/local.stronghold"
-  # stronghold_password: "" <== Should be injected through the env variable `UNICORE__SECRET_MANAGER__STRONGHOLD_PASSWORD`
+  stronghold_password: "secure_password"
```

Comment:
I think we should add a hint to prefer env variables, such as:
```
secret_manager:
  stronghold_password: "secure_password"   <== Consider injecting the password through the env variable `UNICORE__SECRET_MANAGER__STRONGHOLD_PASSWORD` instead
```

Reply:
changed it back :+1: